assets/prototype/domain/eventEditor/containers/queries/useDatetimeIds.js

```diff
@@ -0,0 +1,30 @@
+import * as R from 'ramda';
+import { useApolloClient } from '@apollo/react-hooks';
+import { GET_DATETIMES } from './dates';
+
+const useDatetimeIds = () => {
+	const eventId = R.pathOr(null, ['eeEditorEventData', 'eventId'], window);
```

Comment:
I am not :100: happy with this.
Looking/ open for a better approach.

Reply:
Yeah, great suggestion.
TBD here: https://github.com/eventespresso/event-espresso-core/issues/1955